### PR TITLE
[Feature] Adds filename check

### PR DESCRIPTION
**Pull request by tomasnorre**
This will adds filename check to the JobDSLTest Gradle Task.

It is conform with the Java Identifier conventions which is used by the JobDSL Plugin in Jenkins already.

### CODE REVIEW

File: jenkins-job-dsl-gradle-plugin/src/test/groovy/com/aoe/gradle/jenkinsjobdsl/IncorrectFilenameSpec.groovy [hunk 1]
```diff
@@ -0,0 +1,61 @@
+package com.aoe.gradle.jenkinsjobdsl
+
+import org.gradle.testkit.runner.GradleRunner
+
+import static org.gradle.testkit.runner.TaskOutcome.FAILED
+import static org.gradle.testkit.runner.TaskOutcome.SUCCESS
+
+/**
+ * @author Carsten Lenz, AOE
```
Comment (carstenlenz) [inline jenkins-job-dsl-gradle-plugin/src/test/groovy/com/aoe/gradle/jenkinsjobdsl/IncorrectFilenameSpec.groovy]: No you are not Carsten Lenz ;)

File: jenkins-job-dsl-test-support/src/main/groovy/com/aoe/gradle/jenkinsjobdsl/JobScriptsSpec.groovy [hunk 2]
```diff
@@ -6,6 +6,7 @@ import hudson.model.View
 import javaposse.jobdsl.dsl.*
 import javaposse.jobdsl.plugin.JenkinsJobManagement
 import jenkins.model.Jenkins
+import org.apache.commons.io.FilenameUtils
```
Comment (carstenlenz) [inline jenkins-job-dsl-test-support/src/main/groovy/com/aoe/gradle/jenkinsjobdsl/JobScriptsSpec.groovy]: You should add apache commons io as dependency if you are using it - or do not rely on it

File: jenkins-job-dsl-gradle-plugin/src/test/groovy/com/aoe/gradle/jenkinsjobdsl/IncorrectFilenameSpec.groovy (hunk 1 above)
Comment (tomasnorre) [reply jenkins-job-dsl-gradle-plugin/src/test/groovy/com/aoe/gradle/jenkinsjobdsl/IncorrectFilenameSpec.groovy]: Really.. :) Copy/Paste error.

File: jenkins-job-dsl-test-support/src/main/groovy/com/aoe/gradle/jenkinsjobdsl/JobScriptsSpec.groovy (hunk 2 above)
Comment (tomasnorre) [reply jenkins-job-dsl-test-support/src/main/groovy/com/aoe/gradle/jenkinsjobdsl/JobScriptsSpec.groovy]: I'm not sure I get that ?

**Pull request merged**